src/entities/enum/authorization-type.enum.ts

```diff
@@ -2,4 +2,5 @@ export enum AuthorizationType {
     NO_AUTHORIZATION = "NO_AUTHORIZATION",
     HTTP_BASIC_AUTHORIZATION = "HTTP_BASIC_AUTHORIZATION",
     HEADER_BASED_AUTHORIZATION = "HEADER_BASED_AUTHORIZATION",
+    OAUTH_AUTHORIZATION = "OAUTH_AUTHORIZATION"
```

Comment:
Document this new authorization type [here](https://os2iot.readthedocs.io/en/stage/external-interface-design/external-interface-design.html?highlight=data%20target#fiware).
[Docs repo](https://github.com/OS2iot/OS2IoT-docs/pulls). Point the PR to the stage branch so that it can be tested first.

Reply:
https://github.com/OS2iot/OS2IoT-docs/pull/31